vendor/assets/stylesheets/dvl/core/media.scss

```diff
@@ -21,3 +21,9 @@ img, video {
   width: auto\9;
   height: auto !important;
 }
+
+// Enforcing component consistency
+
+@mixin avatar {
+  border-radius: 50%;
+}
```

Comment:
Ah shit, I missed the most important one: `"ensure_newline_at_eof_on_save": true`